corehq/apps/app_manager/tests/test_suite_multi_select_case_list.py

```diff
@@ -54,11 +54,11 @@ def test_multi_select_case_list(self):
                 </command>
                 <instance id="casedb" src="jr://instance/casedb"/>
                 <session>
-                  <datum id="case_id"
-                         nodeset="instance('casedb')/casedb/case[@case_type='person'][@status='open']"
-                         value="./@case_id"
-                         detail-select="m0_case_short"
-                         detail-confirm="m0_case_long"/>
+                  <instance-datum id="case_id"
```

Comment:
@orangejenny the datum id should change to `selected_cases` for the instance('selected_cases') xpath to work